doc/examples/ml/neon/README.md

```diff
@@ -0,0 +1,48 @@
+# ML pipeline using Nervana Neon and Pachyderm
+
+![alt tag](pipeline.jpg)
+
+This machine learning pipeline integrates Nervana Neon training and inference into a production scale pipeline using Pachyderm.  In particular, this pipeline trains and utilizes a model that predicts the sentiment of movie reviews, based on data from IMDB.
+
+## Deploying Pachyderm
+
+See the [Pachyderm docs](http://docs.pachyderm.io/en/latest/) for details. Note, this demo requires `pachctl` 1.4.0+.  
+
+## Deploying the pipeline
```

Comment:
The user needs to clone this repo or at least pull the files in this example, right? This tutorial assumes they've done that? Maybe it's silly to remind people of that at the beginning of every tutorial.